src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -1170,6 +1170,48 @@ const schema = {
       type: 'array',
       items: { $ref: '#/definitions/secondaryPtsdIncident' },
     },
+    privateMedicalRecordAttachments: {
```

Comment:
Do these new properties need to be added since they're already in the version of `vets-json-schema` we're using here?

Reply:
Not totally. I only put it there for local testing stuff.